PProPanel/ext.js

```diff
@@ -222,11 +222,11 @@ function loadJSX() {
 	var extensionPath = csInterface.getSystemPath(SystemPath.EXTENSION);
 
 	// load general JSX script independent of appName
-	const extensionRootGeneral = extensionPath + '/jsx/';
+	var extensionRootGeneral = extensionPath + '/jsx/';
```

Comment:
Alright, it would certainly refresh this project to a more modern JS standard ;-)

My reason for not including it in this PR: I wanted to focus on the JSX structure change and keep the code style to what is the current standard.

I'd suggest a new PR that contains the switch to const (& maybe 'let'?) where it makes sense. It should update all the JS code to a more modern JS standard that is compliant with the CEP environment.